src/framework/components/text/canvas-font.js

```diff
@@ -197,12 +206,12 @@ Object.assign(pc, function () {
                         ctx = canvas.getContext('2d', {
                             alpha: true
                         });
-                        // Write white text
-                        ctx.fillStyle = this.color;
+                        // Clear the context to transparent
```

Comment:
old comment?

Reply:
I guess I meant "transparent version of the font color", but I'll just remove the `to transparent` part of the comment